### PR TITLE
Remove EspNowNetwork

### PR DESCRIPTION
**Pull request by Johboh**
- `EspNowNetwork` used to be a "fat" library, which was built and published as individual libraries to ESP IDF Component Registry and Platform IO Registry using github actions. This, however, is not compatible with Arduino library registry, as each github repo need to be its own library. The `EspNowNetwork` repo has now been split up and replaced with four individual repositories instead:
  - https://github.com/Johboh/EspNowNetworkHost
  - https://github.com/Johboh/EspNowNetworkHostDriver
  - https://github.com/Johboh/EspNowNetworkNode
  - https://github.com/Johboh/EspNowNetworkShared

All of which are already registered in this registry.

### CODE REVIEW

**Review by per1234 — COMMENTED** [review]
Hello @Johboh. I am confirming your request.

Unlike submissions, this type of request is still handled manually. The Library Manager index maintainer occasionally comes through to clear all the outstanding requests. They will merge this pull request when it is done. You will get a notification from GitHub at that time.

**Review by umbynos — APPROVED** [review]
Done in https://github.com/arduino/library-registry/pull/4976#pullrequestreview-2235204016

**Pull request merged**
